draft-thomson-http-replay.md

```diff
@@ -257,6 +257,14 @@ that the client understands this specification and is willing to retry a request
 in response to a 4NN (Too Early) status code.  A user agent that sends a request
 in early data does not need to include the `Early-Data` header field.
 
+A server that receives a request marked with the Early-Data header field cannot
+wait until the TLS handshake completes in order to render the request safe from
+risk of replay.  The Early-Data header field indicates that early data was used
```

Comment:
"risk of replay" is ambiguous about "risk of this request being a replay" vs. "risk that this request could be replayed elsewhere/elsewhen".  The latter is nonsensical to an informed reader, of course, but we may not want to rely on that.  Take 2 in the comments is better in this regard, but is perhaps at risk of causing "safe" to have too many overloaded meanings.